Source/UnitTests/Common/BitSetTest.cpp

```diff
@@ -70,12 +70,11 @@ TEST(BitSet, Count)
 
 TEST(BitSet, BitOps)
 {
-  BitSet32 a(3), b(5), c;
+  BitSet32 a(3), b(5), c(a);
```

Comment:
I would recommend moving the declaration of `c` down to the first usage instead.